src/index.css

```diff
@@ -25,9 +26,11 @@ a:hover {
 
 body {
   margin: 0;
-  display: flex;
-  place-items: center;
-  min-width: 320px;
+  padding: 0;
+  /* display: flex; */
+  box-sizing: border-box;
+  /* place-items: center; */
+  /* min-width: 320px; */
```

Comment:
remove commented code when no longer needed

Reply:
Ok, I will do that.

Reply:
Done!